libs/auth/src/drivers/magento/login.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { Observable, of, throwError } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { Apollo } from 'apollo-angular';
+
+import { DaffLoginServiceInterface } from '../interfaces/login-service.interface';
+import { DaffLoginInfo } from '../../models/login-info';
+import { DaffAuthToken } from '../../models/auth-token';
+import {
+  MagentoRevokeCustomerTokenResponse,
+  MagentoGenerateTokenResponse,
+  generateTokenMutation,
+  revokeCustomerTokenMutation
+} from './queries/public_api';
+import { DaffMagentoAuthTransformerService } from './transforms/auth-transformer.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoLoginService implements DaffLoginServiceInterface<DaffLoginInfo, DaffAuthToken> {
+  constructor(
+    private apollo: Apollo,
+    public authTransformer: DaffMagentoAuthTransformerService,
+  ) {}
+
+  login({email, password}: DaffLoginInfo): Observable<DaffAuthToken> {
+    return this.apollo.mutate<MagentoGenerateTokenResponse>({
+      mutation: generateTokenMutation,
+      variables: {
+        email,
+        password
+      }
+    }).pipe(
+      map(res => this.authTransformer.transform(res.data.generateCustomerToken))
```

Comment:
How do we accommodate errors like if the username/password doesn't match?

Reply:
I added error handling to the login driver